### PR TITLE
Pass the Payment Action value to Razorpay

### PR DESCRIPTION
**Pull request by seherkdc**
Get the value of payment action configured by the admin `$payment_action`.
Set `$payment_capture` = 0 for "Authorize only" and 1 for "Authorize and Capture".

### CODE REVIEW

File: Controller/Payment/Order.php [hunk 1]
```diff
@@ -36,13 +36,23 @@ public function __construct(
...
+	$this->config = $config;
     }
 
     public function execute()
     {
         $amount = (int) (round($this->getQuote()->getBaseGrandTotal(), 2) * 100);
 
         $receipt_id = $this->getQuote()->getId();
+	    
+	$payment_action = $this->config->getPaymentAction();
```
Comment (ccrims0n) [inline Controller/Payment/Order.php]: indentation

File: Controller/Payment/Order.php [hunk 2]
```diff
@@ -36,13 +36,23 @@ public function __construct(
...
+	    
+	if($payment_action == "authorize") {
```
Comment (ccrims0n) [inline Controller/Payment/Order.php]: please add proper indentation and coding standards

File: Controller/Payment/Order.php (hunk 1 above)
Comment (seherkdc) [reply Controller/Payment/Order.php]: Indentation problem has been fixed. Kindly go ahead with the commit

File: Controller/Payment/Order.php (hunk 2 above)
Comment (seherkdc) [reply Controller/Payment/Order.php]: Fixed the indentation and coding standard problem.

File: Controller/Payment/Order.php [hunk 3]
```diff
@@ -36,6 +36,7 @@ public function __construct(
 
         $this->checkoutFactory = $checkoutFactory;
         $this->catalogSession = $catalogSession;
+	$this->config = $config;
```
Comment (ccrims0n) [inline Controller/Payment/Order.php]: please fix the indentation

File: Controller/Payment/Order.php [hunk 4]
```diff
@@ -44,6 +45,14 @@ public function execute()
 
         $receipt_id = $this->getQuote()->getId();
 
+        $payment_action = $this->config->getPaymentAction();
+
+        if($payment_action == "authorize") {
```
Comment (ccrims0n) [inline Controller/Payment/Order.php]: @seherkdc please update the following
- Space after `if`
- strict comparison
- single quotes around string
- brackets should be in new line

File: Controller/Payment/Order.php (hunk 3 above)
Comment (seherkdc) [reply Controller/Payment/Order.php]: @ccrims0n Indentation fixed.

File: Controller/Payment/Order.php (hunk 4 above)
Comment (seherkdc) [reply Controller/Payment/Order.php]: @ccrims0n Changes Done.

**Review by ccrims0n — APPROVED** [review]

**Pull request merged**